VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/copy-id/copy-id.component.ts

```diff
@@ -9,11 +9,13 @@ import { ClipboardService } from 'ngx-clipboard';
 export class CopyIdComponent implements OnInit {
     @Input() conference: HearingSummary;
     tooltip: string;
+    propertyIdName = 'copy-conference-id';
     @ViewChild('copyID', { static: false }) copyID: ElementRef;
 
     constructor(private clipboardService: ClipboardService) {}
 
     ngOnInit(): void {
+        this.propertyIdName = this.propertyIdName + '-' + this.conference.id;
```

Comment:
There is a slight risk here that if the inputs changed but the component stayed that the id would be wrong. This could happen in the case where we saw the wrong link being copied when one was removed.

I was doing the same and was considering using ngOnChanges instead. I don't think we need to be too concerned with this risk here and if that happened it would signal that there is something more fundamentally wrong with our code. So I think we are fine to leave it as it is.

Just be careful in tests if you call ngOnInit this could append the id multiple times.

Reply:
Would that be related to the same scenario we got where the trackedBy on that for loop was creating issues?